lib/Player.tsx

```diff
@@ -167,7 +165,7 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
       setWaiting(false)
     }, [])
 
-    const onFormat = (format: Format) => {
+    const onFormat = useCallback((format: Format | undefined) => {
```

Comment:
Looks like this can just be inside the `useEffect`, then you don't need a separate `useCallback`?.